pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: code_builder
-version: 3.2.2-dev
+version: 3.2.3-dev
 
 description: >-
   A fluent, builder-based library for generating valid Dart code
 homepage: https://github.com/dart-lang/code_builder
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: ">=2.6.0 <3.0.0"
```

Comment:
please revert this change and the below change

Reply:
Done. I've also reverted this change in https://github.com/dart-lang/code_builder/pull/277